channeldb/kvdb/config.go

```diff
@@ -46,6 +46,8 @@ type EtcdConfig struct {
 
 	Namespace string `long:"namespace" description:"The etcd namespace to use."`
 
+	DisableTLS bool `long:"disabletls" description:"Disable TLS for etcd connection."`
```

Comment:
I think rather than making this a dev only flag as proposed by @cfromknecht, I think it would be enough to just add a sentence here that you should only use this for development.

Reply:
Done

Reply:
We tried this in the past with `--noencryptwallet`, sadly the precedent is that people ignore warnings like this or don't read them.

Personally I'd feel more comfortable placing this footgun behind the `dev` tag, until we have a specific use case that requires it in production.